website/README.md

```diff
@@ -0,0 +1,193 @@
+This website was created with [Docusaurus](https://docusaurus.io/).
+
+# What's In This Document
```

Comment:
I would remove everything bellow this line

Reply:
I deleted the Readme file

Reply:
Still here

Reply:
-1 Let's keep this file. It provides valuable content for maintainers and future committers on how to work on the website. I would recommend removing all things (in the doc and website configs) to do with blogposts as this is a feature we do not want.